source/material-terms.ttl

```diff
@@ -0,0 +1,68 @@
+# Add sameAs to marc enumerations if there is significant usage, need to converge with marcframe mapping.
+
+<Acetat> a :Material ; skos:notation "1001" ; skos:exactMatch rdamat:1001, marcmat:ace .
+<AcrylicPaint> a :Material ; skos:notation "1002" ; skos:exactMatch rdamat:1002, marcmat:acp ; skos:prefLabel "akrylfärg"@sv .
+<Aluminium> a :Material ; skos:notation "1003" ; skos:exactMatch rdamat:1003, marcmat:alu .
+<BristolBoard> a :Material ; skos:notation "1004" ; skos:exactMatch rdamat:1004, marcmat:brb .
+<Canvas> a :Material ; skos:notation "1005" ; skos:exactMatch rdamat:1005, marcmat:can .
+<Cardboard> a :Material ; skos:notation "1006" ; skos:exactMatch rdamat:1006, marcmat:crd .
+<Ceramic> a :Material ; skos:notation "1007" ; skos:exactMatch rdamat:1007, marcmat:cer .
+<Chalk> a :Material ; skos:notation "1008" ; skos:exactMatch rdamat:1008, marcmat:chk ; skos:prefLabel "krita"@sv .
+<Charcoal> a :Material ; skos:notation "1009" ; skos:exactMatch rdamat:1009, marcmat:chr ; skos:prefLabel "kol"@sv .
+<Diacetate> a :Material ; skos:notation "1010" ; skos:exactMatch rdamat:1010, marcmat:dic .
+# http://id.loc.gov/vocabulary/mmaterial/dia see 1046.
+<Dye> a :Material ; skos:notation "1011" ; skos:exactMatch rdamat:1011, marcmat:dye ; skos:prefLabel "färgämne"@sv .
+<Glass> a :Material ; skos:notation "1012" ; skos:exactMatch rdamat:1012, marcmat:gls; ; skos:prefLabel "glas"@sv .
+<Gouache> a :Material ; skos:notation "1013" ; skos:exactMatch rdamat:1013, marcmat:gou ; skos:prefLabel "gouach"@sv .
+<Graphite> a :Material ; skos:notation "1014" ; skos:exactMatch rdamat:1014, marcmat:gra ; skos:prefLabel "grafit"@sv .
+<Hardboard> a :Material ; skos:notation "1015" ; skos:exactMatch rdamat:1015, marcmat:hdb .
+<IllustrationBoard> a :Material ; skos:notation "1016" ; skos:exactMatch rdamat:1016, marcmat:ill .
+<Ink> a :Material ; skos:notation "1017" ; skos:exactMatch rdamat:1017, marcmat:ink ; skos:prefLabel "bläck"@sv .
+<Ivory> a :Material ; skos:notation "1018" ; skos:exactMatch rdamat:1018, marcmat:ivo .
+<Lacquer> a :Material ; skos:notation "1019" ; skos:exactMatch rdamat:1019, marcmat:lac ; skos:prefLabel "lack"@sv .
+<Leather> a :Material ; skos:notation "1020" ; skos:exactMatch rdamat:1020, marcmat:lea .
+<MagneticParticles> a :Material ; skos:notation "1021" ; skos:exactMatch rdamat:1021, marcmat:fer ; skos:prefLabel "magnetiska partiklar"@sv .
+<Metal> a :Material ; skos:notation "1022" ; skos:exactMatch rdamat:1022, marcmat:mtl .
+# http://id.loc.gov/vocabulary/mmaterial/mix no RDA match.
+<Nitrate> a :Material ; skos:notation "1023" ; skos:exactMatch rdamat:1023, marcmat:nit .
+# http://id.loc.gov/vocabulary/mmaterial/nsf no RDA match.
+<OilPaint> a :Material ; skos:notation "1024" ; skos:exactMatch rdamat:1024, marcmat:oip ; skos:prefLabel "oljefärg"@sv .
+<Paper> a :Material ; skos:notation "1025" ; skos:exactMatch rdamat:1025, marcmat:pap ; :sameAs <https://id.kb.se/marc/Paper> .
```

Comment:
👍 We must remove `marc:Paper` (to avoid id conflict in LDDB), remove the subclass `BaseMaterial` (in at least marcframe and deprecate in KBV). And then, map all other "BaseMaterial" marc-enums like this, here in this file (using `sameAs`).

And, of course test this by running MarcFrameConverter + revert and, crucially run a cleanup of all records linking to the marc-enums to re-link them to these terms.

Reply:
Commented out as TODO for now so no accidental merge will break things.

Reply:
We can set this PR as draft if we think we can address this here?

Reply:
Thinking about it I think it could be a good idea to separate the conversion alignments and cleanups into separate PR:s.

Reply:
Yes, this is for defining our sought-after material types, and a *separate* one for "folding" these and various MARC-forms together (FWIW). We might want to "SKOS-map" the MARC-enums here and now if the matches are obvious enough (to capture what we learn), and keep TODO:s about how some of them actually are `owl:sameAs` (like Paper here, which should now be `<Paper> ... ; skos:exactMatch ..., marc:Paper . # TODO: owl:sameAs marc:Paper`).